shell/common/shell_unittests.cc

```diff
@@ -2060,10 +2075,14 @@ TEST_F(ShellTest, DiscardLayerTreeOnResize) {
                static_cast<double>(expected_size.height()));
 
   end_frame_latch.Wait();
+  // Threads not merged.
+  ASSERT_EQ(0, external_view_embedder->GetSubmittedFrameCount());
 
+  end_frame_latch.Wait();
   ASSERT_EQ(1, external_view_embedder->GetSubmittedFrameCount());
```

Comment:
nit: is it possible to assert that threads are merged before this assert?  Same where you have the comment `// Threads not merged`, can that be an assert?

Reply:
Done

Reply:
Actually, we can't assert if threads are not merged here, the threads are merged at the end of the frame. So it is actually merged here.
My comment was misleading, it should say that during the frame, the threads are not merged, they are merged at the end of the frame.
Updated the comment, PTAL